stan/math/prim/fun/identity_constrain.hpp

```diff
@@ -11,12 +11,13 @@ namespace math {
  * transform to the input. This promotes the input to the least upper
  * bound of the input types.
  *
+ * @tparam Jacobian Unused, only here for standardized API
  * @tparam T type of value to promote
  * @tparam Types Other types.
  * @param[in] x object
  * @return transformed input
  */
-template <typename T, typename... Types,
+template <bool Jacobian = false, typename T, typename... Types,
```

Comment:
With a value of false yes we could take all of current ones with `lp` and give them the bool template parameter. Though imo I think this could actually end up with more / confusing code as we have specializations for several of the functions with `lp` as a parameter. I kind of prefer the scheme here where we have a sort of higher level function that will defer to the impls based on the template parameter value.